Documentation/ExtensionArchitecture/HowTo/UpdateExtensions/UpdateWizards/Creation.rst

```diff
@@ -90,26 +94,6 @@ Method :php:`getPrerequisites()`
         }
 
 
-.. index:: Upgrade wizards; Registration
-.. _upgrade-wizards-register:
-
-Registering wizards
-===================
-
-Once the wizard is created, it needs to be registered. Registration is done in
-:file:`ext_localconf.php`:
-
-..  code-block:: php
-    :caption: EXT:my_extension/ext_localconf.php
-
-    $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['ext/install']['update']['myExtension_exampleUpgradeWizard']
```

Comment:
Should we mention that if you need to stay compatible with TYPO3 12 and 11 you can keep this registration?

Reply:
I mentioned it in the versionadded part on top of the page.

Reply:
So I would either put a Version 11 / 12 compatibility section or link to the description in the 11.5 branch of the document